data/data/gcp/iam/outputs.tf

```diff
@@ -0,0 +1,5 @@
+# This simply passes the variable value through this module,
+# but enforces a dependency in the master module.
```

Comment:
It may be worth waiting until we upgrade our terraform version and then just using module dependencies directly.